pybamm/util.py

```diff
@@ -333,7 +333,8 @@ def install_jax(arguments=None):  # pragma: no cover
 
     Command Line Interface:
     -----------------------
-    >>> pybamm_install_jax
+    To install compatible versions of jax and jaxlib, run:
```

Comment:
```suggestion
```

Reply:
Hey @Saransh-cpp, good to see you again.
154743c will update the API Docs